src/library/rekordbox/rekordboxconstants.h

```diff
@@ -0,0 +1,5 @@
+namespace mixxx {
+namespace rekordboxconstants {
+const QString beatsSubversion = QStringLiteral("Rekordbox USB drive");
```

Comment:
I am unsure about that. Is the place where we have found the data relevant? 
I think it is the Rekordbox version or found feature that has created the beat grid is relevant. 
Can we distinguish that?

Reply:
OK